ingestion/functions/template.yaml

```diff
@@ -0,0 +1,22 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: AWS Lambda functions for epid ingestion
+
+Resources:
+  RetrievalFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: retrieval/
+      Handler: retrieval.lambda_handler
+      Runtime: python3.8
+      Description: Retrieve the executing machine IP
+      # Function's execution role
+      Policies:
+        - AWSLambdaBasicExecutionRole
+        - AWSLambdaReadOnlyAccess
+      Timeout: 10 # Seconds
+
+Outputs:
+  RetrievalFunction:
+    Description: "Retrieval Lambda function ARN"
+    Value: !GetAtt RetrievalFunction.Arn
```

Comment:
Not sure what that means, can you comment please?

Reply:
Commented. Not clear to me that this section will ultimately be used -- but adding while I set the resources up to see if it's of any help (I'll remove it later if not).